### DISCUSSION TITLE
Language bar options - Docked on taskbar greyed out (W11)

### DISCUSSION THREAD

**Discussion by luchris** [valinet/ExplorerPatcher#1518] [General]
ExplorerPatcher version 2261.1555.55.1

May i know is the greyout caused by the patcher or windows?

**Comment by pyrates999** [comment]
Please show screen shot and include windows build.

**Comment by luchris** [comment]
![Win11](https://github.com/valinet/ExplorerPatcher/assets/13183475/3b9b3cf1-1ff6-47fd-8fc1-d6ba3993d44a)

Edition	Windows 11 Pro
Version	22H2
Installed on	‎29/‎3/‎2023
OS build	22621.1778
Experience	Windows Feature Experience Pack 1000.22642.1000.0

**Comment by pyrates999** [comment]
You have a preview update installed: https://support.microsoft.com/en-us/topic/may-24-2023-kb5026446-os-build-22621-1778-preview-3c547100-7a73-4ae6-bb7d-ebd02e87dc04

You need to uninstall it due to symbols not being available to download for previews and insider builds:

https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Comment by luchris** [comment]
Thanks for pointing out. Will uninstall it and come back with the result

**Comment by luchris** [comment]
Edition	Windows 11 Pro
Version	22H2
Installed on	‎29/‎3/‎2023
OS build	22621.1702
Experience	Windows Feature Experience Pack 1000.22641.1000.0

Does Windows Feature Experience Pack has any impact? Not working still

**Comment by pyrates999** [comment]
Uninstall explorer patcher and reinstall it.  Then check that windows symbols have been downloaded:

https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Comment by luchris** [comment]
Uninstalled, restart PC, reinstalled and restarted. Win+x brings up the menu. Still not working for me

**Comment by pyrates999** [comment]
After you've uninstalled explorer patcher and rebooted, does **text services and input languages** no longer have the option **docked in the taskbar** greyed out?  Are you able to select it?

**Comment by luchris** [comment]
Hi, yes "Docked in the taskbar" is still greyed out. Even though symbols are installed.

![image](https://github.com/valinet/ExplorerPatcher/assets/13183475/1c6cb744-ea55-40bc-8f8f-6bea0ec8c7e5)

**Comment by pyrates999** [comment]
Ok, but my last comment asked you to uninstall explorer patcher, reboot, and then check in **text services and input languages**, is the option **docked in the taskbar** greyed out? Are you able to select it?
